ci/scripts/connect-release-init.js

```diff
@@ -0,0 +1,163 @@
+const child_process = require('child_process');
+const path = require('path');
+const fs = require('fs');
+
+const { checkPackageDependencies } = require('./check-npm-dependencies');
+
+const args = process.argv.slice(2);
+
+if (args.length < 1) throw new Error('Check npm dependencies requires 1 parameter: semver');
+const [semver] = args;
+
+const allowedSemvers = ['patch', 'minor'];
+if (!allowedSemvers.includes(semver)) {
+    throw new Error(`provided semver: ${semver} must be one of ${allowedSemvers.join(', ')}`);
+}
+
+const ROOT = path.join(__dirname, '..', '..');
+
+const exec = (cmd, params) => {
+    console.log(cmd, ...params);
+
+    const res = child_process.spawnSync(cmd, params, {
+        encoding: 'utf-8',
+        cwd: ROOT,
+    });
+    if (res.status !== 0) {
+        console.log(cmd, ...params);
+        console.log(res);
+    }
+    return res;
+};
+
+const commit = ({ path, message }) => {
+    exec('git', ['add', path]);
+    exec('git', ['commit', '-m', `${message}`]);
+};
+
+const comment = ({ prNumber, body }) => {
+    exec('gh', ['pr', 'comment', `${prNumber}`, '--body', body]);
```

Comment:
Maybe it would be nice to have a sanity check at the beginning of the script that checks that this dependency is available in the system.
I assume that would always be available in `.github/workflows` where this is going to be run. But not sure if some day we want to run this locally.

Reply:
sure, it would be nice but I see this as a nice to have.